### PR TITLE
Nice small changes

### PR DESCRIPTION
**Pull request by Vukky123**
5b22875 - 👥 RewardHeaderBar now displays Donate Now if the reward is a donation.
* ~~I attempted to change Order Summary to Donation Summary and Pay With Salad to Donate With Salad for donations too, however, I couldn't figure it out. Does a dev know how to do this?~~ [see this comment](https://github.com/SaladTechnologies/salad-applications/pull/498#issuecomment-680052473)

87d0560 - Added Origin 👥, Xbox, and Apple as platforms to RewardInfoPanel and RewardPlatform.

Changes marked with :busts_in_silhouette: are @stabbed's idea!
*This is my first code contribution. Do not be surprised if this PR is horrible.*

### CODE REVIEW

File: packages/web-app/src/modules/reward-views/components/RewardHeaderBar.tsx [hunk 1]
```diff
@@ -111,6 +111,7 @@ class _RewardHeaderBar extends Component<Props> {
   render() {
     const { reward, classes, ...rest } = this.props
 
+    let donation = reward?.tags.includes('donations')
```
Comment (seniorquico) [inline packages/web-app/src/modules/reward-views/components/RewardHeaderBar.tsx:114]: The tags are not always the same case. Instead we should match against the `category` which will remain consistent:

```suggestion
    let donation = reward?.category === "donation"
```

**Comment by seniorquico** [comment]
Thanks, @Vukky123. The Salad Pay labels will have to remain as-is for technical and branding purposes. Otherwise, it looks good to me aside from the one requested change. If you get this updated, I can run it through QA testing.

**Comment by seniorquico** [comment]
@Vukky123 Is this ready for QA testing?

**Comment by Vukky123** [comment]
Should be ready now.

**Pull request merged**